bundles/EcommerceFrameworkBundle/CoreExtensions/ClassDefinition/IndexFieldSelection.php

```diff
@@ -336,4 +329,26 @@ public function setWidth($width)
     {
         $this->width = (int)$width;
     }
+
+    public function getParameterTypeDeclaration(): ?string
+    {
+        return '?\\Pimcore\\Bundle\\EcommerceFrameworkBundle\\CoreExtensions\\ObjectData\\IndexFieldSelection';
```

Comment:
@weisswurstkanone Imho `return sprintf('?%s', IndexFieldSelection::class); ` would be nicer, wdyt?